src/analyzer.ts

```diff
@@ -72,14 +73,78 @@ function getFullWarningMessage(warning: Warning): string {
   }`;
 }
 
-export class StreamAnalyzer extends Transform {
-  config: ProjectConfig;
+/**
+ * A stream that tells the BuildAnalyzer to resolve each file it sees. It's
+ * important that resolving loaded files happens as soon as possible, in a way
```

Comment:
I tried to imply this with the second half of the sentence, but will make it more explicit that blocking analysis and other tasks is very, very bad :)